cloudprem/kubernetes.tf

```diff
@@ -72,10 +81,10 @@ resource "kubernetes_config_map" "dozuki_resources" {
     "buckets.json" = <<-EOF
       {
         "default": {
-          "guide-images": "${module.guide_images_s3_bucket[0].this_s3_bucket_id}",
-          "guide-pdfs": "${module.guide_pdfs_s3_bucket[0].this_s3_bucket_id}",
-          "documents": "${module.documents_s3_bucket[0].this_s3_bucket_id}",
-          "guide-objects": "${module.guide_objects_s3_bucket[0].this_s3_bucket_id}"
+          "guide-images": "${local.guide_images_bucket}",
+          "guide-pdfs": "${local.guide_pdfs_bucket}",
+          "documents": "${local.documents_bucket}",
+          "guide-objects": "${local.guide_objects_bucket}"
```

Comment:
I don't see where we're specifying these locals. Maybe we could split these resources out into their own file? Just to get a little better locality?

Reply:
The local definitions are in the main.tf at the top. This whole thing is undesirable from an organization perspective so any recommendations are welcome.

Reply:
Hmm. We could split out a top-level file for the configmap resources. But that doesn't really help locality. Though it might be a nice organization tweak. Maybe we can split out the S3 resources into a separate file with it?
https://github.com/Dozuki/CloudPrem-Infra/blob/c90fea4d76dfa60f5a3cca8bc8c7cccc59fb5b9c/cloudprem/main.tf#L121-L145
Or the configmap?
https://github.com/Dozuki/CloudPrem-Infra/blob/c90fea4d76dfa60f5a3cca8bc8c7cccc59fb5b9c/cloudprem/kubernetes.tf#L29-L161

That configmap resource could totally be in a file by itself.

I dont really know though. I dont think TF makes locality or organization very easy. This is probably fine for now...